receptor-python-worker/pyproject.toml

```diff
@@ -1,3 +1,20 @@
 [build-system]
 requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
+
+[project]
+authors = [
+    {email = "info@ansible.com"},
+]
+description = "The receptor-python-worker command is called by Receptor to supervise the operation of a Python worker plugin."
+dynamic = [
+    "dependencies",
+    "scripts",
+    "version"
+]
+name = "receptor-python-worker"
+
+[project.optional-dependencies]
+dev = [
```

Comment:
FWIW, extras is not the best place to put contributor-facing deps. Extras are public API for the end-users. Do you want to expose `pip install receptor-python-worker[dev]` to every arbitrary person on the internet?
Perhaps, when the dependency groups PEP is accepted, this config file will be usable for this dep category, but not today. Today, it's best to describe your dev env as a set of requirements outside the package metadata.